CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ include(CTest)
 include(CMakeDependentOption)
 
 # ----------------------------- Setup -----------------------------
+find_program(CCACHE_PROGRAM ccache)
```

Comment:
do you see better performance with ccache over incremental `make`? Just curious

Reply:
It's much faster if you continuously rebuild the project.